kubernetes/bin/README.md

```diff
@@ -0,0 +1,2 @@
+`/bin` is used to store binary of main.go. `make manager` will build manager binary and we can run `./bin/manager` to test run operator locally.
```

Comment:
Can be merged with the Instructions.md file so there are two types of documents: one for the end user and another for developers.

Reply:
Sure. One more question, this folder just have one manager binary there and I think it should be added to .gitignore. We can not keep this folder in git if it's empty.

Are you ok to exclude `/bin` in git?

Reply:
I think I will just remove `bin` folder. When we build binary, `/bin/manager` will be untracked.